ouroboros-consensus-diffusion/test/consensus-test/Test/Consensus/PeerSimulator/Trace.hs

```diff
@@ -70,72 +53,30 @@ mkChainSyncClientTracer ::
 mkChainSyncClientTracer tracer =
   Tracer $ \case
     TraceRolledBack point ->
-      trace $ "Rolled back to: " ++ condense point
+      trace $ "Rolled back to: " ++ tersePoint point
     TraceFoundIntersection point _ourTip _theirTip ->
-      trace $ "Found intersection at: " ++ condense point
+      trace $ "Found intersection at: " ++ tersePoint point
     _ -> pure ()
   where
     trace = traceUnitWith tracer "ChainSyncClient"
 
+prettyTime :: MonadMonotonicTime m => m String
+prettyTime = do
+  Time time <- getMonotonicTime
+  let ps = diffTimeToPicoseconds time
+      milliseconds = (ps `div` 1_000_000_000) `mod` 1_000
+      seconds = (ps `div` 1_000_000_000_000) `rem` 60
+      minutes = (ps `div` 1_000_000_000_000) `quot` 60
+  pure $ printf "%02d:%02d.%03d" minutes seconds milliseconds
```

Comment:
> The arithmetic here looks wrong.

My bad. I failed to count 0s properly :) I edited my comment now.

Reply:
The code in question has proven to work quite fine so far, it is only moved as part of this PR. I am fine with your other rendering; it is maybe a bit less verbose than the current implementation.

As for `div`/`mod` vs. `quot`/`rem`, I don't really know what happened but I agree that it is confusing. Since we're only handling positive numbers, it doesn't really matter, and `quot`/`rem` is then more efficient. Fine if I use that pair consistently?

Reply:
Fine with me to use either of the pairs. Why is quot/rem more efficient?

Reply:
(Not that that kind of performance consideration has any impact at our level.)

Reply:
Done in 3f85ddc619d23ea634725c59cee5794604507b59.